sass/pages/_assets.scss

```diff
@@ -18,29 +18,109 @@
         margin-bottom: 52px;
     }
 
-    .card {
-        margin: 0;
-        height: 8rem;
-    }
+    .asset-card {
```

Comment:
I would move `asset-card` into it's own file `sass/components/_asset-card.scss`.
Pages folder (`pages/*`) is kind-of legacy; where both BEM and non-BEM gets mixed.

Reply:
Sure, I'm still new to BEM so wasn't sure how file structure should work so I just updated the existing file. I'll do that tomorrow.